lib/stripe/subscriptions/subscription.ex

```diff
@@ -73,6 +79,7 @@ defmodule Stripe.Subscription do
                :customer => Stripe.id() | Stripe.Customer.t(),
                optional(:application_fee_percent) => integer,
                optional(:billing) => String.t(),
+               optional(:billing_cycle_anchor) => Stripe.timestamp(),
```

Comment:
Would you mind adding all 3 to the `update` spec as well?

Reply:
No problem! It's ready for another look.